Octokit.Reactive/Clients/IObservableMigrationClient.cs

```diff
@@ -0,0 +1,13 @@
+﻿namespace Octokit.Reactive
+{
+    public interface IObservableMigrationClient
+    {
+        /// <summary>
+        /// A client for GitHub's Migrations API
+        /// </summary>
+        /// <remarks>
+        /// See the <a href="https://developer.github.com/v3/migration/migrations/">Enterprise License API documentation</a> for more information.
+        /// </remarks>
+        IObservableMigrationsClient Migration { get; }
```

Comment:
agree that this is (the only one ive seen so far) a weird naming thing in github API that makes it hard for us to follow our rules.  But the convention test will fail if you dont have the same property name in `IObservableMigrationClient` that you used in `IMigrationClient`